browser-test/src/applicant/northstar_application_version_fast_forward.test.ts

```diff
@@ -1723,7 +1741,7 @@ class FastForwardApplicantActor {
   getCardListLocator(
```

Comment:
**suggestion (non-blocking):** I'm loving all these helper functions.  I think this one and the ApplicationStatusCardGroupName enum could be reused across tests.  What would you think of putting them in `applicant_questions.ts` or starting a new test support file for the landing page?

Reply:
Oh now your poking on my desire to rebuild better supporting classes :D 

I'm not going to move this right now. Let's chat next week. I don't want to bake in the names of things if they don't align with new northstar designs. I want everything to match, but I'm not certain I know what we officially call them.